app/models/manageiq/providers/amazon/inventory/parser/cloud_manager.rb

```diff
@@ -224,6 +224,21 @@ def cloud_databases
     end
   end
 
+  def normalize_db_status(raw_status)
+    case raw_status
+    when "available"
+      "available"
+    when "creating", "starting"
```

Comment:
Should we add `modifying` here or maybe to `unavailable`?

Reply:
Yeah I didn't add a few yet because I wanted to check what the availability of the database is (e.g. I'm assuming that `"backing-up"` is an online backup).  Similarly I'm assuming that `"modifying"` would mean unavailable but want to try to find that out first.

Reply:
ah ok that makes sense